src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.utils.Color;
+
+public class Circle extends Figure {
+    protected double radius;
+
+    public Circle(Color color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public double getRadius() {
+        return radius;
+    }
+
+    public void setRadius(double radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle, "
+                + "area: " + Math.round(calculateArea() * 10) / 10
+                + " sq. units, " + "radius: " + radius
+                + " units, color: " + getColor().toString().toLowerCase());
```

Comment:
![image](https://github.com/mate-academy/jv-oop-advanced/assets/20326858/2875ae28-f8af-4a46-ab14-3edd02867284)

Reply:
Not fixed